editor/block-mover/style.scss

```diff
@@ -3,7 +3,8 @@
 	top: 10px;
 	left: 0;
 	padding: 0 14px 20px 0; // handles hover area
-
+	z-index: 100;
```

Comment:
Works great 👍 but we should probably use our `z-index` mixin. https://github.com/WordPress/gutenberg/blob/master/editor/assets/stylesheets/_z-index.scss#L16